index.html

```diff
@@ -10024,38 +10062,51 @@ <h4 id="Beast Shape - Aquatic Animal">Beast Shape - Aquatic Animal</h4>
                <td>Water Breathing
                </td>
                <td>When in water, you have Adv (+) on all CON checks and don’t need to hold
+                  your breath.  On land, you have DisAdv (-) on all CON checks and need to hold
                   your breath.
                </td>
                <td>Self
                </td>
                <td>Pass
                </td>
             </tr>
+            <tr>
+               <td>Core
+               </td>
+               <td>Creature from the Depths
+               </td>
+               <td>When in water, all attacks of opportunity against you have DisAdv (--).
+               </td>
+               <td>Self
+               </td>
+               <td>Pass
+               </td>
+            </tr>
             <tr>
                <td>Core
                </td>
                <td>Water Strike
                </td>
-               <td>Storm forward and strike with great force. Choose a point you can see
-                  within 20ft and speed to that tile, you then may immediately make a regular
-                  melee attack against a target. On a hit the target takes additional Light damage
-                  for each tile moved (distance in ft divided by 5). 15ft is equal to 3 tiles, 3
-                  additional damage is dealt. You cannot use Water Strike on a target within 5ft.
+               <td>Storm forward and strike with great force. Choose a creature you can see
+                  within 20ft and speed directly to them and make a regular
+                  melee attack against the creature dealing DEX damage. On a hit the creature takes additional Light damage
+                  for each tile moved (distance in ft divided by 5). 15ft is equal to 3 tiles, 3*Light (Heavy)
+                  additional damage is dealt. 
```

Comment:
Same issues as other abilities commented on above. Also why 2 AP? I guess it is technically not as good as Savage Leap any more but is it infinite use? That's way too good yo